cmd/kops/rollingupdatecluster.go

```diff
@@ -137,6 +150,7 @@ func RunRollingUpdateCluster(f *util.Factory, out io.Writer, options *RollingUpd
 			return fmt.Errorf("cannot build kube client for %q: %v", contextName, err)
 		}
 
+		// TODO use k8s client here so that we can be backwards compat
```

Comment:
So we need to start respecting https://github.com/kubernetes/kops/issues/1262

Our comments are moving towards documentation. So we need to focus on things like complete (and correctly spelled) sentences. With correct punctuation, etc.. 

Please update this line so that it reads correctly.